skimage/morphology/_watershed.pyx

```diff
@@ -10,6 +10,8 @@ All rights reserved.
 Original author: Lee Kamentsky
 """
 import numpy as np
+from libc.math cimport sqrt
```

Comment:
I personally did not use this sqrt, but some contributor needed this to calculate the euclidean distance in line 41: 

https://github.com/ThomasWalter/scikit-image/blob/66743dc3c344b0f3c8bc60fd45d49ea465fe0784/skimage/morphology/_watershed.pyx#L41

I changed the place of the import for PEP8 reasons.